www/site/content/docs/chef-workstation/chef-run-users-guide.md

```diff
@@ -185,16 +180,15 @@ preparation for converging the target node. When running on that node the
 `first` recipe finds its local dependency on the `pretty_simple` cookbook and
 then runs its `second` recipe.
 
-You can specify different cookbook sources in `Policyfile.rb`. [Private
-supermarket documentation](https://docs.chef.io/config_rb_policyfile.html)
+You can specify different cookbook sources in `Policyfile.rb`, including a private supermarket. See the [Policyfile documentation](https://docs.chef.io/config_rb_policyfile.html) for examples of how to do this.
 
 ## Connecting to Automate 2
 
 You can configure remote nodes managed with `chef-run` to send run
 information to Automate. First, [generate an auth
 token](https://automate.chef.io/docs/admin/#creating-a-standard-api-token).
 
-Next, add the token to [config.toml](TODO: link to configuration page),
+Next, add the token to [config.toml](../reference/config),
```

Comment:
Is this the right way to do a local reference?

Reply:
We're going to do a final TODO pass to clean up the links.  https://gohugo.io/content-management/cross-references/ has the info for link format.